### ISSUE TITLE
Property assertion asserts.has ( ) doesn't work

### ISSUE THREAD

**Issue by AnidemDex** [AlexDarigan/WAT-GDScript#210]
If you try to use `asserts.has()` with any object type, it will stop the excecution.

**Error:**
![image](https://user-images.githubusercontent.com/7025991/116833099-fc2c7b00-ab7c-11eb-8a44-4a1ca8b4228c.png)
**Using:**
![image](https://user-images.githubusercontent.com/7025991/116833125-12d2d200-ab7d-11eb-98d9-615e7a7aca22.png)

Maybe I'm using it wrong?

**Comment by AnidemDex** [comment]
As a workaround i use:
```gdscript
asserts.is_true("name" in Node.new())
```

**Comment by AlexDarigan** [comment]
This is for Arrays and Dictionaries. Not for objects.

**Comment by AnidemDex** [comment]
> This is for Arrays and Dictionaries. Not for objects.

@AlexDarigan still the same issue using
```gdscript
asserts.has("echo", {"echo":""})
# or
asserts.has("echo", ["echo"])
```
I'm using the lastest version in AssetsLib

**Comment by AlexDarigan** [comment]
It seems there is a minor syntax error where its Property.has.new() instead of Property.has()

**Comment by AlexDarigan** [comment]
Should be resolved by cda4868afc5cc93beb4212ef14778f0e50c98319
